### DISCUSSION TITLE
`int64-strategy` JSON Schema emitter option

### DISCUSSION THREAD

**Discussion by clemensv** [microsoft/typespec#3278] [General]
I'm missing a better discussion in the docs of "widely interoperable" in the [`int64-strategy`](https://typespec.io/docs/libraries/json-schema/reference/emitter#int64-strategy) option of the JSON Schema emitter. I'm aware of the discussion of precision in [RFC8259, Section 6](https://www.rfc-editor.org/rfc/rfc8259#section-6) and, thus, `Number.MAX_SAFE_INTEGER`, but that discussion assumes that an implementation chooses to store all numbers as an IEEE754 64-bit floating point number. Is there evidence for this being indeed such a common case that users are being nudged into representing any "long" as string by a "widely interoperable" claim?

**Comment by bterlson** [comment]
JavaScript has one of the most widely deployed JSON implementations in the world and is limited to doubles only.
